leakcanary-android/build.gradle

```diff
@@ -2,6 +2,7 @@ apply plugin: 'com.android.library'
 
 dependencies {
   api project(':leakcanary-analyzer')
+  compile 'com.android.support:support-fragment:27.0.1'
```

Comment:
should be `api`.

Reply:
Also, can the dependency can be compileOnly? Do a Class.forName check before using the FragmentRefWatcher?
The extra dep doesn't really matter for a dev tool, though!